config/locales/en.yml

```diff
@@ -371,11 +371,18 @@ en:
       push_heading: Push Notifications
   webauthn_verifications:
     expired_or_already_used: The token in the link you used has either expired or been used already.
+    no_port: No port provided. Please try again.
     prompt:
       title: Authenticate with Security Device
       authenticating_as: Authenticating as
       authenticate: Authenticate
       no_webauthn_devices: You don't have any security devices enabled
+    successful_verification:
+      title: Success!
+      body: Please close this browser.
```

Comment:
Can we rename the translation so it better reflects the translation itself rather than where it is used for? eg. `close_browser`? (Same goes for the failure case)